components/movementsensor/wheeledodometry/wheeledodometry.go

```diff
@@ -357,6 +369,10 @@ func (o *odometry) trackPosition(ctx context.Context) {
 			o.position.X += (centerDist * math.Cos(o.orientation.Yaw))
 			o.position.Y += (centerDist * math.Sin(o.orientation.Yaw))
 
+			distance := math.Hypot(o.position.X, o.position.Y)
```

Comment:
[q] were distances reported in meters? I may need to divide by another factor to get the right lat/long coordinates, they seem large just from testing. 0.05 degs lat/long incremented from the tests.

Reply:
yeah it's meters

Reply:
bit confused about why there's issues tho... not sure why putting it into coordinates is changing the scale

Reply:
So the conversion from meters to lat long is [weird ](https://www.usna.edu/Users/oceano/pguth/md_help/html/approx_equivalents.htm) and non-linear. Essentially the opposite of this [formula](https://en.wikipedia.org/wiki/Mercator_projection#Derivation)

This library is nice because it gives us a way to convert given the distance away from 0,0 and the angle the x,y point is at. However it needs a distance in km.